apps/dashboard/app/models/files.rb

```diff
@@ -55,16 +55,16 @@ def self.can_download_as_zip?(path, timeout: Configuration.file_download_dir_tim
     error = nil
 
     if ! (path.directory? && path.readable? && path.executable?)
-      error = "You can only download a directory as zip that you have read and execute access to"
+      error = I18n.t('dashboard.files_directory_download_unauthorized')
     else
       # Determine the size of the directory.
       o, e, s = Open3.capture3("timeout", "#{timeout}s", "du", "-cbs", path.to_s)
 
       # Catch SIGTERM.
       if s.exitstatus == 124
-        error = "Timeout while trying to determine directory size"
+        error = I18n.t('dashboard.files_directory_size_calculation_timeout')
       elsif ! s.success?
-        error = "Error with status #{s} when trying to determine directory size: #{e}"
+        error = I18n.t('dashboard.files_directory_size_unknown', exit_code: s, error: e)
```

Comment:
`files_directory_size_calculation_error_message` to match `files_directory_size_calculation_timeout`